Added titles to manager side pages and fixed grid for item details page

LawreenL opened the pull request:
**Finishing up for the Demo:**
- Added the titles "Reservations" and "Loans" to manager side pages. 
- Modified for loops to if statements so the "Reservations" title changes to "No Reservations" when empty (same with Loans). 
- Modified the grid layout for item details page to add padding on top and making it more responsive

*Loans page with active Loans*
<img width="1006" alt="screen shot 2018-10-28 at 5 54 17 pm" src="https://user-images.githubusercontent.com/15279043/47622408-1e8c3c00-dadb-11e8-94fb-efbf56af5abb.png">
*Loans page with no active Loans*
<img width="1029" alt="screen shot 2018-10-28 at 5 54 31 pm" src="https://user-images.githubusercontent.com/15279043/47622407-1e8c3c00-dadb-11e8-9158-b7a6375c185e.png">
##
*Item Details page on desktop*
<img width="1031" alt="screen shot 2018-10-28 at 5 53 25 pm" src="https://user-images.githubusercontent.com/15279043/47622410-1f24d280-dadb-11e8-8b3b-7dc8a3e3b64b.png">
*Item Details page on mobile*
<img width="405" alt="screen shot 2018-10-28 at 5 53 50 pm" src="https://user-images.githubusercontent.com/15279043/47622409-1e8c3c00-dadb-11e8-9515-75b1ac6a7500.png">

LawreenL committed:
Added titles to manager side pages and fixed grid for item details page

StephenOrJames commented on dam/loans/templates/loans/allReturns.html:
```diff
@@ -1,24 +1,26 @@
+                </div>
+            {% endfor %}
+        {% else %}
             <h1 style="text-align: center;">No Loans</h1>
```
Could you remove the alignment from this? All `h1` are aligned center from `style.css`.

StephenOrJames commented on dam/loans/templates/loans/allReservations.html:
```diff
@@ -2,22 +2,24 @@
+                </div>
+            {% endfor %}
+        {% else %}
             <h1 style="text-align: center;">No Reservations</h1>
```
Could you remove the alignment from this? All `h1` are aligned center from `style.css`.

StephenOrJames reviewed (dismissed):
A thought: would it be better to have the heading be the same, whether not there are items to be listed, and if there are no content then have an additional message underneath the title (rather than have the title change)? I'd personally prefer that (rather than just having a page with just a title), but up to you.

<hr>

Also, looks like the item detail page is slightly misaligned on mobile, and I assume that has to do with the rows and columns are nested (`row > row > col` vs `row > row > col > row` if I'm reading this correctly) since you added an "extra" row (there's already a row in `base.html`).

A solution might be to move the row in base out to each page individually (and maybe even the container), but that can be considered outside the scope of this fix unless you also want to handle that here as well.

![image](https://user-images.githubusercontent.com/8732346/47622798-b0964380-dadf-11e8-97b9-1c3e8b943ca0.png)

LawreenL committed:
edited base.html to add top padding and removed top container from manager pages, also removed styling from h1 tags

LawreenL commented:
Added Subtitles below bcs I can't decide but i think it looks good now.
<img width="1029" alt="screen shot 2018-10-28 at 7 03 16 pm" src="https://user-images.githubusercontent.com/15279043/47623167-2b615d80-dae4-11e8-81f3-bb2032f6d7ff.png">

For the grid, ended up adding the top padding to base since it is used for every page and removed the top div from manager pages- leaving the grid alignment for @lawzeem

LawreenL committed:
added subtitle to allreservations and allloans

StephenOrJames commented on dam/loans/templates/loans/allReservations.html:
```diff
@@ -2,22 +2,23 @@
+            {% endfor %}
+        {% else %}
+            <h1>Reservations</h1>
+            <h6 style="text-align: center;">No active reservations right now.</h6>
```
Maybe `p` instead of `h6`?

StephenOrJames commented on dam/loans/templates/loans/allReturns.html:
```diff
@@ -1,24 +1,25 @@
+            {% endfor %}
+        {% else %}
+            <h1>Loans</h1>
+            <h6 style="text-align: center;">No active loans right now.</h6>
```
Maybe `p` instead of `h6`?

StephenOrJames reviewed (approved):
Excess whitespace at the top of pages that have a header is something to consider revisiting in the future (i.e. making it more consistent with those that don't have a header or something), but I'm okay with this.
